Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu
+
+WORKDIR /performance-dashboards
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install necessary libraries for subsequent commands
+RUN apt-get update && apt-get install -y podman dumb-init python3.6 python3-distutils python3-pip python3-apt
```

Comment:
Why do we need all of this?, only python3 and python3-requirements should be necessary

Reply:
Removed unnecessary dependencies and kept only minimal set of packages.